frontend/src/actions/resources.js

```diff
@@ -78,7 +89,7 @@ const checkSymptoms = withStory((story) => {
   ].includes(story.sick);
 
   const isCurrentlyOnUS = {
-    predicate: baseCondition && story.currentLocation === countries.USA,
+    predicate: baseCondition && story.country === countries.USA,
```

Comment:
It does not work for city picker since when the user is using city picker, the country field will be autofilled with "United States". However, `countries.USA` is defined as "United States of America" here. Therefore, it should work for both "United States of America" as the user uses the widget and for "United State" as the user uses city picker.